moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -83,14 +83,9 @@ class SceneTransforms : public robot_state::Transforms
   }
 
 private:
-  bool knowsObject(const std::string& id) const
+  bool knowsFrame(const std::string& id) const
```

Comment:
add inline comments about why its Frame now and not Object

Reply:
It's because this function seemed to be used to check if the transform to a frame is known, rather than to check if the object exists in the world, but I think there is a use case for both. Either way, before changing the name the original function should be deprecated. I readded both.

Reply:
Correction: I changed knowsObject slightly so it returns true when the object is known to the world (named frames are ignored). The previous version of the function makes more sense as "knowsFrame". I am not sure what I think of only checking the world for an object, and ignoring the attachedBody objects, but that's another discussion.